.travis.yml

```diff
@@ -24,6 +24,26 @@ jobs:
     after_success:
     - CODECOV_NAME=coverage.out bash <(curl -s https://codecov.io/bash)
 
+  # Build the Contour container image with the local Docker. Create a kind
+  # cluster, push the image we built into it, and verify that both Contour
+  # and Envoy come up. This is an initial placeholder as we experiment with
+  # different approaches to integration testing.
+  - name: integration
+    install:
+    - mkdir -p $HOME/bin
+    - ./hack/travis/install-kubernetes-toolchain.sh
+    - make container IMAGE=docker.io/projectcontour/contour VERSION=ci
```

Comment:
Maybe make this obvious that it's not a real container? Like contour:ci?

Reply:
The fully qualified container name this generates is `docker.io/projectcontour/contour:ci`. Is that similar to what you were thinking of?

Reply:
No my thought is to make it not look like a real image. I realize the CI system won't have any creds to push images, but by using image names that couldn't match a real image, there's no risk in pulling an image by mistake because it matches an existing one.

Reply:
e.g. `make container IMAGE=contour`

Reply:
I can change that, but I still need to tag with the official names so I can push them into the Kind cluster.

Reply:
Why do you need official names? Just load the image and use from the k8s objects. Shouldn't need to push to a real registry.

Reply:
I'm not modifying the example YAML, so I push the images into Kind to match the names that it uses. When we can move all that to kustomize, we can generate YAML with rewritten image names.

Reply:
Ok then. Maybe this just needs a design doc like @youngnick started previously to understand the work. This is fine, ship it.😀